.gitignore

```diff
@@ -52,6 +52,7 @@ docs/src/templates/breadcrumbs.html*
 cms-django.iml
 integreat-cms.iml
 webpack-stats.json
+.pytest_cache
```

Comment:
Why do you add this to the `.gitignore`? I never noticed `.pytest_cache` before, but having a quick look at my system it seems that pytest automatically generates another `.gitignore` inside it which is also recognized by the top level git repository – making this change redundant.

Reply:
Hi Peter. Thank you for your review!
Just checked, indeed this .pytest_cache is not required. Removed. I added it while switching my branch to python 3.11